Gemfile.lock

```diff
@@ -342,6 +344,7 @@ GEM
     zeitwerk (2.6.12)
 
 PLATFORMS
+  aarch64-linux-musl
```

Comment:
This line / file isn't changed in the other Dockerfile refactor PRs - should it be?

Reply:
You got me :D I've been finding that whenever I run `docker compose build` this line gets added to my `Gemfile.lock` file, so I thought I would save me having to stash that every time by sneaking it in 😅 

In the other repos the platform `aarch64-linux` has been present for 2 years now (see forms-runner for example [[1](https://github.com/alphagov/forms-runner/blame/c2744035f052c866ee0ab313688d8a4db0bc3eaf/Gemfile.lock#L466)]. I'm not sure why this repo got missed.

Or, now that I look at it, why for forms-api it's `aarch64-linux-musl` and not just `aarch64-linux` 🤔